src/deploy.sh

```diff
@@ -509,10 +516,70 @@ function kernel_deploy()
   fi
 }
 
+# Deploy a package
+#
+# @flag How to display commands. See `cmd_manager` inside `src/kwlib.sh`
+# @pkg_type The type of package
+# @target VM, local or remote?
+# @remote_raw Address of remote target
+#
+function pkg_deploy()
```

Comment:
I'm not sure if I like this approach because we are duplicating code from `kernel_install` and complicating how we read the code. I agree that we can have a specific function for deploying packages in a remote machine but just for it.